frontend/src/App.tsx

```diff
@@ -119,6 +120,10 @@ const App = (): ReactElement => {
               path="/profile/:profileId"
               component={() => <ProfilePage user={user} setUser={setUser} />}
             />
+            <Route
+              path="/savedapartments/:profileId"
```

Comment:
Change the path to `path="/saved"` without `profileId`, we should not expose the id here, all info should be in user state and be retrieved from then.